### DISCUSSION TITLE
GitHub token name in github_actions.yaml

### DISCUSSION THREAD

**Discussion by eveyp** [ropensci/targets#554] [Ideas]
Could the name of the GitHub access token in `inst/templates/github_actions.yaml` be switched from `GITHUBPAT` to `GITHUB_TOKEN`?

`GITHUB_TOKEN` seems to be what many of the `usethis` templates call it so this would make it easier to work with an existing access token (it took me a while to realize this was why my targets action kept failing).

It seems that most of the [examples in rlib/actions](https://github.com/r-lib/actions/blob/master/examples) use `GITHUB_TOKEN`.

**Comment by wlandau** [comment]
Sure, @eveyp. Looks like [every repo automatically gets a `GITHUB_TOKEN`](https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/), so that sounds reasonable. Should be updated now.
